doc/tech/release_notes/release_process.md

```diff
@@ -1,23 +1,53 @@
-[<< Back](../)
-# Release Process
+# Common NFVI Telco Taskforce Technical Release Process
 
+## Table of Contents
+* [Scope](#scope)
+* [Release Model](#releasemodel)
+* [Roadmap](#roadmap)
+* [Events](#events)
+* [Release Sign-off](#signoff)
 
-## Contents
-* [Overview](#overview)
-* [H1](#h1)
-* [H2](#h2)
-* [H3](#h3)
 
+<a name="scope"></a>
+## Scope
 
-<a name="overview"></a>
-## Overview
+The scope of this document is to define the release management and its process for planning, scheduling and controlling the build, in addition to proof reading and release deployment. This document defines the release model by organising the git branches and managing the code and other artefacts efficiently in well-structured format.
+It should be considered a living document until it is agreed and signed by all the parties.
 
+<a name="releasemodel"></a>
+## Release Model
 
-<a name="h1"></a>
-## H1
+As a guiding principle, all the development occurs in “Master” branch. All the contribution for the milestone (especially M3) goes into ”Master”. After M4 (Proof Reading), at one point in the release candidate (RC) (for now maintain RC0) for Baldy and based on experience we shall increase or limit what get into the final release. To make this happen, branch out from “Master”, create a delivery branch (“Baldy”,”Hallasan”, “Baraque”, etc). Developers can continue their next release branch work in “Master”. At the end of each release, artifacts are “tagged” in GitHub according to the guideline and principles defined.
 
-<a name="h2"></a>
-## H2
+To deliver a hot fix (also known as patch) into the latest release, simply apply the hot fix in the delivery branch and tag it (4.0.1-Baldy). Once the hot fix is delivered, cherry pick technique will be applied to carry over the changes into the current release. As shown in the below diagram.
```

Comment:
as discussed on the called, I'd like to propose to add a sentence here or rephrase: the mind set should always be to fix a problem on master first - irrespective of existing release branches. So, by default a hotfix should go to master first. Only if it does not apply anymore, a "local" fix on a release branch is valid.

Reply:
Be aware that we are treating documentation as code, which is an imperfect analogy.

Reply:
I remember in the weekly call that we discussed and concluded to deliver the hot fix (also called patch) into the latest available release (which is Baldy in this case). To perform the patch delivery, we simply perform the hot fix in the delivery branch, and tag it (4.0.1-Baldy) as per the community standard practice and guideline. I remember the same pattern is followed in ONAP as well. Once the hot fix is delivered (and everyone back to happiness), we will decide how to carry over the changes into the current release branch. Most probably cherry pick technique will apply (same shown in the figure). Only during the stabilization phase, all the fixes will go in "master" and cherry pick technique will apply for taking some of the fixes onto delivery branch. Hot fix process will be more the exception than the rule.

Reply:
@gkunz - I remember in the weekly call that we discussed and concluded to deliver the hot fix (also called patch) into the latest available release (which is Baldy in this case). To perform the patch delivery, we simply perform the hot fix in the delivery branch, and tag it (4.0.1-Baldy) as per the community standard practice and guideline. I remember the same pattern is followed in ONAP as well. Once the hot fix is delivered (and everyone back to happiness), we will decide how to carry over the changes into the current release branch. Most probably cherry pick technique will apply (same shown in the figure). Only during the stabilization phase, all the fixes will go in "master" and cherry pick technique will apply for taking some of the fixes onto delivery branch. Hot fix process will be more the exception than the rule.

Reply:
Hi @prabhubalan. I am referring to best practices established in mature upstream communities, such as OpenStack. According to the OpenStack documentation [1] on stable branches, all fixes must exist on master first "*unless the change simply does not make sense on master*" and are then backported to stable branches. Please see [2] (item 4) and [3].

I do see that we are handling documentation as code here - as pointed out by @bfcohen - so we can be a bit more relaxed about not "breaking" a stable release. However, some ways-of-working could still apply.

[1] https://docs.openstack.org/project-team-guide/stable-branches.html
[2] https://docs.openstack.org/project-team-guide/stable-branches.html#appropriate-fixes
[3] https://docs.openstack.org/project-team-guide/stable-branches.html#processes

Reply:
I agree with @gkunz.  It makes sense to 
1. refer to established practices in a mature organization and 
1. patch master first, when possible, and cherry-pick from there.  Documentation fixes may wind up going to multiple active releases, and merge history may get confusing if there is a mesh of cherry-picking going on.
1. be pragmatic about the differences in code vs. documentation.

Reply:
Hi @gkunz , @kedmison - I have updated them now. Merge / Approve if you are happy with the content.